lib/json/jwe.rb

```diff
@@ -240,7 +240,7 @@ def verify_cbc_authentication_tag!
       expected_authentication_tag = OpenSSL::HMAC.digest(
         sha_digest, mac_key, secured_input
       )[0, sha_size / 2 / 8]
-      unless secure_compare(authentication_tag, expected_authentication_tag)
+      unless ActiveSupport::SecurityUtils.secure_compare(authentication_tag, expected_authentication_tag)
```

Comment:
Why calling `ActiveSupport::SecurityUtils` directly here?

Reply:
Good catch, missed that before pushing :o